ydb/core/tx/datashard/datashard_ut_write.cpp

```diff
@@ -52,7 +50,7 @@ Y_UNIT_TEST_SUITE(DataShardWrite) {
 
         Cout << "========= Read table =========\n";
         {
-            auto tableState = TReadTableState(server, MakeReadTableSettings("/Root/table-1")).All();
+            auto tableState = ReadTable(server, shards.at(0), "table-1", tableId, 1);
```

Comment:
maybe unit_assert_values(shards.size(), values)
or cycle in readtable over shards?

Reply:
I like the second option: ReadTable iterates over all the shards.